docker-compose.yml

```
@@ -77,7 +77,7 @@ services:
       - "5672"
 
   redis:
-    image: redis:3-alpine
+    image: redis:5-alpine
```

This is a bump of two major versions, but [Redis release cycle](https://redis.io/topics/releases) specifies they _try very hard to make the Redis API mostly backward compatible_ and _upgrading to newer versions is usually trivial_. 4.0 (blog post)[https://redislabs.com/blog/redis-4-0-0-released/]/[ml post](https://groups.google.com/d/msg/redis-db/5Kh3viziYGQ/58TKLwX0AAAJ) and 5.0 [blog post](https://redislabs.com/blog/redis-5-0-is-here/)/[ml post](https://groups.google.com/d/msg/redis-db/l0OXDAlwosU/JyYfeQpyBwAJ) doesn’t any breaking change either. So it is probably fine but Redis 3 is still in production at the moment, see https://github.com/metabrainz/docker-redis.

That would be nice to:
1. Have more detailed commit message, e.g. by including the above rationale;
2. Allow setting Redis version through a Docker environment variable for tests (before changing the default), e.g.

```suggestion
    image: redis:${REDIS_VERSION:-3}-alpine
```

(untested suggestion)